src/Components/test/testassets/BasicTestApp/FormsTest/InputRangeComponent.razor

```diff
@@ -0,0 +1,16 @@
+﻿<p>
+This component renders two range inputs:
+One with the 'value' attribute specified before the 'min' and 'max' attributees, and one specified after.
+</p>
+
+<p>
+This verifies that the value is applied after all other attributes so it doesn't get incorrectly clamped
+by the default 'min' and 'max' attribute values.
+</p>
+
+<input id="range-value-first" value="@(210)" type="range" min="20" max="220" />
+<input id="range-value-last" type="range" min="20" max="220" value="@(210)" />
+
+@code {
+
```

Comment:
```suggestion
     private const int RangeValue = 210;
```

It's a bit unusual to see literals specified using C# constants. I don't want someone to accidentally clean that up and break what this is testing.